portfolio/src/main/webapp/script.js

```diff
@@ -15,7 +15,9 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
+async function addRandomGreeting() {
```

Comment:
You fetch the greeting from the servlet at /hello, but you don't add that response to the greetingContainer.

Action item: assign the textFromResponse variable to the greetingContainer

Reply:
fixed it, going to recommit